.gitignore

```diff
@@ -1,2 +1,6 @@
 /target
 /bot_token.txt
+.dart_tool/
```

Comment:
These are not related to this fix, and imo do not belong on master. Can you revert this?